model/task/results.go

```diff
@@ -133,3 +134,38 @@ func FilterTasksOnStatus(tasks []Task, statuses ...string) []Task {
 
 	return out
 }
+
+type Result struct {
+	Name  string `json:"name"`
```

Comment:
as brian said, these should really be bson tags, I'm also unclear why this needs to be exported

Reply:
done.

Reply:
I don't see that change

Reply:
This intermediate struct is no longer used. The db returns Stats (which have bson tags) and those go straight into APIStats (which has json tags)